packages/tools/client-debug-view/src/components/AudienceView.tsx

```diff
@@ -187,7 +205,7 @@ function HistoryView(props: HistoryViewProps): React.ReactElement {
 				},
 			}}
 		>
```

Comment:
How does this growing list respond to vertical screen space? Do we get a scrollbar, or does it just extend off the bottom? If possible, it would be nice to ensure this list gets a scrollbar when it exceeds its vertical space.

Reply:
<img width="290" alt="image" src="https://user-images.githubusercontent.com/111468570/206536815-3545b298-e2b3-4b1c-b1b0-58f2a12e3901.png">


Made it `scrollable`

Reply:
Sorry, I actually meant just the `history` block. I think we always want to see the tabs menu and the active audience members view above, but if the history is long, letting us scroll that section would be useful.

Reply:
It's probably also best to continue using `auto` rather than `scroll` for the overflow policy - that way we only see a scrollbar when the list actually overflows its vertical space.

Reply:
<img width="289" alt="image" src="https://user-images.githubusercontent.com/111468570/206590510-dc73d87c-10a2-4486-b5a2-9a553619aefe.png">

Changed 😄

Reply:
That looks great! Thanks!